server/handlers/cdn-caching.js

```diff
@@ -1,3 +1,31 @@
+/**
+ * How quintype handles toggle between different CDN providers
+ * - The default CDN provider is set as `Cloudflare` and can be toggled to the required CDN provider (for now we have support for `akamai`, in future we can support multiple CDN providers based on requirement).
+ *
+ *   How to toggle between cdn providers:
+ *   - If your app is cloned from malibu then, in `malibu/app/server/app.js` pass `cdnProvider` with required value to `isomorphicRoutes`(No need to pass if `cloudflare` is your cdn provider).
```

Comment:
This again, you have to mention ppl to contact quintype before they do the change